components/AppHeader.vue

```diff
@@ -2,28 +2,36 @@
   <header class="container header clearfix" role="banner">
     <div class="title-area">
       <nuxt-link to="/" class="logo-link">
-        <img class="logo" src="/logo--unocha.svg" alt="Office for the Coordination of Humanitarian Affairs">
+        <img class="logo" src="/logo--unocha.svg" :alt="$t('United Nations Office for the Coordination of Humanitarian Affairs')">
       </nuxt-link>
 
       <h1 class="title" v-if="title">{{ title }}</h1>
-      <h1 class="title" v-else>Situation Reports</h1>
-      <span class="subtitle" v-if="title">Situation Report</span>
-      <span class="subtitle" v-else>United Nations Office for the Coordination of Humanitarian Affairs</span>
-      <span class="last-updated" v-if="updated"><span class="viz--480">Last </span> updated: <time :datetime="updated">{{ $moment(updated).format('YYYY-MM-DD') }}</time></span>
+      <h1 class="title" v-else>{{ $t('Situation Reports') }}</h1>
+      <span class="subtitle" v-if="title">{{ $t('Situation Report') }}</span>
+      <span class="subtitle" v-else>{{ $t('United Nations Office for the Coordination of Humanitarian Affairs') }}</span>
+      <span class="last-updated" v-if="updated">{{ $t('Last updated') }}: <time :datetime="updated">{{ $moment(updated).locale(locale).format('DD MMM YYYY') }}</time></span>
```

Comment:
another benefit: with locale actually being tracked in the store, we can localize dates according to language. moment has built-in locale support and I already had french included.